doc/doxygen/design.dox

```diff
@@ -33,11 +33,17 @@
  * allocation and management of device-specific memories including those found
  * in accelerators. In terms of communication APIs, UCT defines interfaces for
  * immediate (short), buffered copy-and-send (bcopy), and zero-copy (zcopy)
- * communication operations. The short operations are optimized for small
- * messages that can be posted and completed in place. The bcopy operations
- * are optimized for medium size messages that are typically sent through a
- * so-called bouncing-buffer. Finally, the zcopy operations expose
- * zero-copy memory-to-memory communication semantics. 
+ * communication operations.
+ *
+ * \b Short: This type of operations is optimized for small messages that can be posted and completed
```

Comment:
In this and the bcopy and zcopy paragraphs please change "This type of operations" to "This type of operation"